roles/systemd_service_manager/README.md

```diff
@@ -0,0 +1,83 @@
+systemd_service_manager
+========================
+
+This Ansible role manages systemd services on target hosts. It enables, disables, and creates new systemd services based on the provided configuration.
+
+Requirements
+------------
+N/A
+
+Role Variables
+--------------
+
+| Variable          | Required | Default |                                        Comments                                               |
+|-------------------|----------|---------|-----------------------------------------------------------------------------------------------|
+| `systemd_units`   |   yes    |   N/A   | A list of dictionaries containing configurations for systemd services. See below for details. |
```

Comment:
I'm happy to have it merged but from naming perspective.
The PR references systemd services and also systemd units, both terms are used through the code. My advice would be to use one or the other (preferably unit which is more general allowing us to create all kind of systemd entities).